recipes/django-rq/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "django-rq" %}
+{% set version = "0.9.6" %}
+{% set sha256 = "368051f0a3ef08670dd186ea0202cbc7644e199fd48fd31eace2e637e915a7f7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    - rq >=0.5.5
+
+test:
+  commands:
+    - python -c "import django; from django.conf import settings; settings.configure(INSTALLED_APPS=['django_rq'], RQ_QUEUES={}); django.setup(); import django_rq"    
+
+about:
+  home: https://github.com/ui/django-rq
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

Comment:
`LICENSE.txt` is already included in the source file, so please remove it from the recipe and change to just `LICENSE.txt`. :)